core/src/replay_stage.rs

```diff
@@ -1906,7 +1903,18 @@ impl ReplayStage {
         debug!(
            "after replayed into bank, updated cost model instruction cost table, current values: {:?}",
            cost_model.read().unwrap().get_instruction_cost_table()
-       );
+        );
+        update_cost_model_time.stop();
+
+        inc_new_counter_info!("replay_stage-update_cost_model", 1);
+        datapoint_info!(
+            "replay-loop-timing-stats",
+            (
+                "update_cost_model_elapsed",
+                update_cost_model_time.as_us() as i64,
+                i64
+            )
+        );
```

Comment:
Please use `ReplayTiming` for these.

Reply:
This one will be taken care of when my other PR being merged.
In PR#18123, I added a thread to `replay_stage` as service thread to handle slow/expensive ops, such as all cost model related stuff. While doing that, I added `replayServiceTiming` to collect stats form that thread. When merge, I will move this timing to `ReplayServiceTiming`.

Reply:
I would then suggest to add these metrics in the same PR, or one the other one is already merged in.